src/content/en/updates/2018/10/background-fetch.md

```diff
@@ -0,0 +1,448 @@
+or totally fails.
+
+You can get a list of all the active background fetches using `getIds`:
+
+    navigator.serviceWorker.ready.then(async (swReg) => {
+      const ids = await swReg.backgroundFetch.getIds();
+    });
+
+### Background fetch registrations
+
+A `BackgroundFetchRegistration` (`bgFetch` in the above examples) has the following:
+
+<table class="responsive">
+  <tbody>
+    <tr><th colspan=2>Properties</th></tr>
+    <tr>
+      <td><code>id</code></td>
+      <td><code>string</code><br>The background fetch's ID.</td>
+    </tr>
+    <tr>
+      <td><code>uploadTotal</code></td>
+      <td><code>number</code><br>The number of bytes to be sent to the server.</td>
+    </tr>
+    <tr>
+      <td><code>uploaded</code></td>
+      <td><code>number</code><br>The number of bytes successfully sent.</td>
+    </tr>
+    <tr>
+      <td><code>downloadTotal</code></td>
+      <td><code>number</code><br>The value provided when the background fetch was registered, or
+      zero.</td>
+    </tr>
+    <tr>
+      <td><code>downloaded</code></td>
+      <td><code>number</code><br>The number of bytes successfully received.
+      <p>This value may decrease. For example, if the connection drops and the download cannot be
+      resumed, in which case the browser restarts the fetch for that resource from scratch.</p></td>
+    </tr>
+    <tr>
+      <td><code>result</code></td>
+      <td><p>One of the following:</p>
+        <ul>
+          <li><code>""</code> - The background fetch is active, so there's no result yet.</li>
+          <li><code>"success"</code> - The background fetch was successful.</li>
+          <li><code>"failure"</code> -  The background fetch failed. This value only appears when
+          the background fetch totally fails, as in the browser cannot retry/resume.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>failureReason</code></td>
+      <td><p>One of the following:</p>
+        <ul>
+          <li><code>""</code> - The background fetch hasn't failed.</li>
+          <li><code>"aborted"</code> – The background fetch was aborted by the user, or
+          <code>abort()</code> was called.</li>
+          <li><code>"bad-status"</code> - One of the responses had a not-ok status, e.g. 404.</li>
+          <li><code>"fetch-error"</code> - One of the fetches failed for some other reason, e.g.
+          CORS, MIX, an invalid partial response, or a general network failure for a fetch that
+          cannot be retried.</li>
+          <li><code>"quota-exceeded"</code> - Storage quota was reached during the background
+          fetch.</li>
+          <li><code>"download-total-exceeded"</code> - The provided `downloadTotal` was
+          exceeded.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>recordsAvailable</code></td>
+      <td><code>boolean</code><br>Can the underlying requests/responses can be accessed.</td>
```

Comment:
Would it be worthwhile to clarify that this applies to `match()` and `matchAll()`?